index.md

```diff
@@ -195,96 +129,113 @@ either the result of a digital migration or information that enables the creatio
 of an emulation environment to render a representation. The result of this
 operation is the creation of a new version of the AIP.
 
-The result of a new version of the AIP is stored separately from the submission
-as explained in detail in section [5.2.2](#aipreps).
-
-Furthermore, the AIP format allows updating the AIP by adding a new version of
-the submission. This allows supporting the AIP edition which is defined in OAIS
-as follows:
+Furthermore, the AIP format allows updating the AIP which correspond to the AIP 
+edition which is defined in OAIS as follows:
 
 > “AIP  Edition: An  AIP  whose  Content  Information  or Preservation
 > Description  Information  has been upgraded or improved with the intent not to
 > preserve information, but to increase or improve it. An AIP edition is not
 > considered to be the result of a Migration. [@OAIS2012, p. 1-9]”
 
-The result of an AIP Edition is stored as part of the submission as explained in
-detail in section [5.2.1](#aipcontsubm).
+### Generation
 
-## Cardinality of the SIP to AIP transformation
+If the logical AIP is changed, the physical representation of the information 
+in a container may change as well. We call a *generation* of the AIP a new 
+manifestation of physical container files representing a new version of a 
+logical AIP. The new generation of the AIP can be a single container file or a 
+set of container files.
 
-Regarding the transformation of SIPs to AIPs, in the OAIS it is stated that "one
-or more SIPs are transformed into one or more Archival Information Packages (AIPs)
-for preservation [@OAIS2012, p. 2-8]”.
+A generation of the AIP is created as a consequence of preservation policy 
+decisions. These are generally applied across AIPs without reference to their 
+source or the content contained in them. Examples:
 
-The recommendation in this regard is to guarantee a one-to-one relationship
-between the logical SIP and the logical AIP whereas the relation of the
-corresponding physical packages of SIPs and AIPs can be any kind of many-to-many
-mapping.
+- Changes to the information package structure to conform to a new specification 
+- Replacing all PDF v1.4 -> 2.0s with PDF/A equivalents due to archival policy change
```

Comment:
We should review these defs together to make sure that
- they say what's intended
- they're easy to understand
- the differences between them are clear.